pyproject.toml

```
@@ -41,7 +41,7 @@ nendo = "^0.1.2"
 pydantic = { version = "^2.0.0, <2.5.0" }
 pydantic-settings = "^2.1.0"
 pyrubberband = "^0.3.0"
-essentia = "^2.1b6.dev1110"
+BeatNet = "^1.1.3"
```

```suggestion
BeatNet = "^1.1.3"
PyAudio = "^0.2.14"
```

From the BeatNet repo: 
> Note: Before installing the BeatNet make sure [Librosa](https://librosa.org/) and [Madmom](https://madmom.readthedocs.io/en/latest/installation.html) packages are installed. Also, pyaudio is a python binding for Portaudio to handle audio streaming. If Pyaudio is not installed in your machine, depending on your machine type either install it thorugh pip (Mac OS and Linux) or download an appropriate version for your machine (Windows) from [here](https://www.lfd.uci.edu/~gohlke/pythonlibs/). Then, navigate to the file location through commandline and use the following command to install the wheel file locally: